.github/workflows/run-bats-tests.yml

```diff
@@ -49,3 +50,38 @@ jobs:
       - name: Execute Test Cases
         run: |
           REPO_DIR="./" CLEAN_UP_TEST_ENVIRONMENT=true ./bats/lib/bats/bin/bats ./bats/tests
+
+      # Send the result to Slack when a PR is merged and tests were executed regularly
+      - name: Notify success
+        if: ${{ success() && contains(fromJson('["push", "schedule"]'), github.event_name) }}
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: ${{ secrets.CHANNEL_ID_FOR_MESSAGES }}
+          # If https://github.com/slackapi/slack-github-action/issues/84 is fixed, payload.json can be shared among the following two steps (DRY).
+          payload: |
+            {
```

Comment:
According to https://github.com/slackapi/slack-github-action#usage, a message payload can be defined as a JSON file, but it doesn't work now as I wrote in L.60. So similar JSONs are repeated.